google-cloud-container/lib/google/cloud/container.rb

```diff
@@ -90,13 +90,13 @@ module Cloud
     module Container
       # rubocop:enable LineLength
 
-      FILE_DIR = File.realdirpath(Pathname.new(__FILE__).join("..").join("container"))
+      FILE_DIR = File.realdirpath Pathname.new(__FILE__).join("..").join("container")
```

Comment:
These changes are just going to be overwritten on the next generation, aren't they?

Reply:
No, I added `rubocop -a` to the actual synth process. Your other comment points out a good example of why this can be problematic. I thought it would at least be interesting.